WordPress/gutenberg
# Auto-refresh Popovers

## Description
The `Popover` components are now refreshed every 500ms in order to recalculate their size or position.

Fixes WordPress/gutenberg#11194.

## How has this been tested?
Manually following these steps:
1. Activate the tips on More > Options > Enable tips
2. Switch to the Fullscreen Mode without dismissing the tip
3. Make sure the tip is placed under the '+' icon (see After screenshot below).

## Screenshots 
**Before**
<img width="1262" alt="screen shot 2018-10-29 at 11 05 36" src="https://user-images.githubusercontent.com/1233880/47643318-5fcc2c80-db6b-11e8-84c6-06028885cb79.png">

**After**
<img width="1270" alt="screen shot 2018-10-30 at 19 22 34" src="https://user-images.githubusercontent.com/1233880/47740881-38638580-dc79-11e8-99fa-dd4983975a29.png">

## Types of changes
Bug fix

## Checklist:
